test/bond/test/james.cljc

```diff
@@ -80,6 +80,37 @@
                  (target/quuk 1)))
     (is (= [6 5] (target/quux 8 7 6 5)))))
 
+(defn two-arg-fn [arg1 arg2] nil)
+
+(defmulti some-multimethod {:arglists '([arg1 arg2 arg3])} :dispatch-fn)
```

Comment:
Functions to be stubbed/spied should be defined in the `bond.test.target` namespace